deployment/build-and-deploy-images.ps1

```diff
@@ -41,13 +41,30 @@ az acr build --image datagatewaymodule:$deploymentId --registry $acrName --file
 az acr build --image simulatedtemperaturesensormodule:$deploymentId --registry $acrName --file Distributed.IoT.Edge.SimulatedTemperatureSensorModule/Dockerfile .
 Set-Location -Path $deplymentDir
```

Comment:
Should we change deplymentDir to deploymentDir?

Reply:
Nice catch!